tests/e2e/config.go

```diff
@@ -39,6 +39,14 @@ type e2eTestConfig struct {
 		User string `gcfg:"user"`
 		// vCenter password in clear text.
 		Password string `gcfg:"password"`
+		// vmc vCentre cloudadmin username
+		VmcCloudUser string `gcfg:"vmc-cloudadminuser"`
```

Comment:
why can't we use the existing username and password parameter

Reply:
This test is specific to VMC cloud admin and devops user.